docs/developer/develop-minikube.md

```diff
@@ -0,0 +1,178 @@
+* [Gemini Code Assist + Google Cloud Code](https://marketplace.visualstudio.com/items?itemName=GoogleCloudTools.cloudcode)
+
+Restart VSCode. Once VSCode has restarted let's install the development dependencies (minikube, skaffold and kubectl). We will let the Google Cloud Code extension manage the dependencies and they will be installed in a seperate folder, not in your system folders. In VSCode:
+ * Control-Shift-P (or Command-Shift-P on Macs) to open the command menu
+ * Choose `Cloud Code:  Control Minikube`
+ * Choose `Continue with Managed dependencies`
+
+This will have VSCode install the dependencies (minikube, skaffold and kubectl) and it can take a while.
+
+![Cloud Code Dependencies Install](../images/cloudcode-dependencies.png)
+
+### Start Minkube cluster
+We will now start the minikube k8s cluster
+ * Control-Shift-P (or Command-Shift-P on Macs) to open the command menu
+ * Choose `Cloud Code:  Control Minikube`
+ * Choose `Start` 
+
+![Minikube Start](../images/cloudcode-startminikube.png)
+
+![Minikube Start Output](../images/cloudcode-minikubestart.png)
+
+NOTE: If you want to change the default cluster CPU and Memory usage, you can set those before starting the cluster
+
+    $ minikube config set cpus 4
+    $ minikube config set memory 16384
+
+### Turbinia source and deployment code 
+Now we have VSCode setup we are going to get a copy of the Turbinia source and deployment code.
+Clone the [Turbinia repository](https://github.com/google/turbinia) by forking the Turbinia repository into your own Github account and clone it locally from there.
+
+    $ git clone ssh://git@github.com:[YOUR_GITHUB_ACCOUNT]/turbinia.git
+
+Let's get the helm charts for the Turbinia deployment. In your cloned turbinia repository
+
+    $ mkdir charts && cd charts
+    $ helm repo add osdfir-charts https://google.github.io/osdfir-infrastructure
+    $ helm pull osdfir-charts/turbinia --untar && cd ..
+
+Open up the turbinia folder in VSCode.
+
+### Prepare Cluster
+Open a terminal (inside VSCode is the easiest, but any terminal will do) and let's configure skaffold, the local cluster and the additional helm repository for Redis.
+
+    $ skaffold config set --global local-cluster true
+    $ eval $(minikube -p minikube docker-env)
+    $ helm repo add bitnami https://charts.bitnami.com/bitnami
+
+### Verify Setup
+Execute a build with skaffold (from the root of the cloned Turbinia Github repository)
+
+    $ skaffold build
+
+This will build a Turbinia Server container image succesfully if skaffold has been correctlty setup and configured as described above.
+
+![Skaffold Build](../images/cloudcode-skaffoldbuild.png)
+
+### Install the Turbinia Client
+We will install the Turbinia client into a Python virtual environment to be able to control Turbinia during our development workflow.
+
+    $ python -m venv .venv (or use your favorite virtual env manager)
+    $ ./venv/bin/activate
+    $ pip install turbinia-client
```

Comment:
Any reason not to use the version under development?

Reply:
100%, my bad. Changed to use local dev version.